supply/lib/supply/uploader.rb

```diff
@@ -25,11 +25,21 @@ def perform_upload
 
       upload_binaries unless Supply.config[:skip_upload_apk]
 
+      promote_track if Supply.config[:track_promote_to]
+
       UI.message("Uploading all changes to Google Play...")
       client.commit_current_edit!
       UI.success("Successfully finished the upload to Google Play")
     end
 
+    def promote_track
```

Comment:
Is it possible to reuse the code [here](https://github.com/sschendel/fastlane/blob/promote_track/supply/lib/supply/uploader.rb#L143)?

Can you summarize how the promote to track option is different than calling `supply` with a different track than is currently set in google play? I just want to make sure that this feature is neccessary and not something that can already be done with the existing set up 👍